sdk/location/src/main/java/com/klaviyo/location/KlaviyoLocationManager.kt

```diff
@@ -288,7 +304,19 @@ internal class KlaviyoLocationManager : LocationManager {
             }
         }
 
-        geofences.map { geofence ->
+        // Filter to nearest 20 if we have more than 20 geofences
+        // TODO: In Phase 3, integrate with getCurrentLocation() from PR #371 to get precise location
```

Comment:
I thought phase 3 would be just the changes to how the fetch geofences API call works, to include the location in query params.
I think it'd be nice to see the new filtering method in use in this PR for local filtering